### DISCUSSION TITLE
🥲❌🩻🗣️HELP MEEEE PLEASE stupid wrong pnpm version and I can't update for some stupid reasonnnnnnnnnnnnn

### DISCUSSION THREAD

**Discussion by yousmellstupidhaha** [HasangerGames/suroi#358] [Q&A] [answered]
![Screenshot 2024-07-05 104710](https://github.com/HasangerGames/suroi/assets/111159381/4cc587c7-d31a-461a-9f7e-20870cd3484d)
Somebody help me please... 🥲 COME ON SOMEONE HEOLP MEEEE

**Comment by yousmellstupidhaha** [comment]
Cry Cry

**Comment by yousmellstupidhaha** [comment]
GG @kaklikOf13

**Comment by yousmellstupidhaha** [comment]
LOL

**Comment by yousmellstupidhaha** [comment]
Sorry about the name

**Comment by yousmellstupidhaha** [comment]
🤡

**Comment by juni-purr** [comment]
since when did the brainrot from the suroi server get to the repository

also use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack use corepack

**Reply by yousmellstupidhaha** [reply]
![Screenshot 2024-07-09 214639](https://github.com/HasangerGames/suroi/assets/111159381/a4c0fbc7-1139-48d0-bafe-fdad50fe7fd9)

**Comment by Compositr** [comment, marked as answer]
I agree with kenos use corepack is comes with all sane versions of nodejs

`corepack prepare pnpm@VERSION --activate`

**Reply by yousmellstupidhaha** [reply]
Uh... 
![image](https://github.com/HasangerGames/suroi/assets/111159381/5f1a2f64-ee8e-43dd-a035-00eb8fb3cd28)

**Reply by Compositr** [reply]
No Lmao replace VERSION with the version of PNPM you want such as 9.5.0

**Reply by yousmellstupidhaha** [reply]
💀why am I retarded

**Reply by yousmellstupidhaha** [reply]
Still problems! YAY
![rect1-8](https://github.com/HasangerGames/suroi/assets/111159381/a654cb93-85a9-47f2-af2f-77e20d278d06)

**Reply by Compositr** [reply]
There needs to be a space between the version number and --activate

**Reply by yousmellstupidhaha** [reply]
Ugh! my dear sir, I found that out last night. 
All it said though was: "Preparing pnpm@9.5.0 for immediate activation..." . the when I tried `pnpm install` , it still had the same problem. Am I meant to do something else then `pnpm install` ? or am I totally missing something. 

Best regards.
-Wedgiebuttpoop

**Reply by Compositr** [reply]
Try running `corepack enable` and then reopening your terminal window

**Reply by Compositr** [reply]
Then do the same pnpm install command again

**Reply by yousmellstupidhaha** [reply]
Ok woowwee! Thank you so much! I hope sometime I might be able to maybe help with suroi someday! maybe...
Thanks!

**Comment by DamienVesper** [comment]
switch to c

**Reply by yousmellstupidhaha** [reply]
![Screenshot 2024-06-10 135457](https://github.com/HasangerGames/suroi/assets/111159381/b2220acb-8fbe-4cf3-b7d0-c3b6259ebe14)

**Comment by noahdunnagan** [comment]
bad

**Reply by yousmellstupidhaha** [reply]
what

**Comment by yousmellstupidhaha** [comment]
what is going on here
![image](https://github.com/HasangerGames/suroi/assets/111159381/1259237b-aec5-415d-ad4e-6bc72bf69864)

**Reply by yousmellstupidhaha** [reply]
![image](https://github.com/HasangerGames/suroi/assets/111159381/6b7ac1c4-2749-4c8d-ba51-dbe41e92a517)
